project/MimaBuild.scala

```diff
@@ -86,7 +86,7 @@ object MimaBuild {
 
   def mimaSettings(sparkHome: File, projectRef: ProjectRef): Seq[Setting[_]] = {
     val organization = "org.apache.spark"
-    val previousSparkVersion = "3.0.0"
+    val previousSparkVersion = "3.1.1"
```

Comment:
This is intentionally the first version of 3.1 line. We don't have 3.1.0 and we don't need to change at every 3.1.x release.